# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../eterm16/lvalue-assignment.patch.Eterm
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- Eterm-0.9.3/src/command.c.orig	2005-11-23 20:03:25.411195268 +0100
+++ Eterm-0.9.3/src/command.c	2005-11-23 20:06:11.729851499 +0100
@@ -3002,9 +3002,9 @@
     Xfd = XConnectionNumber(Xdisplay);
     D_CMD(("Xfd = %d\n", Xfd));
     cmdbuf_ptr = cmdbuf_endp = cmdbuf_base;
-    AT_LEAST((int) num_fds, Xfd + 1);
+    AT_LEAST(num_fds, Xfd + 1);
     if (pipe_fd >= 0) {
-        AT_LEAST((int) num_fds, pipe_fd + 1);
+        AT_LEAST(num_fds, pipe_fd + 1);
     }
     if ((cmd_fd = command_func(argv)) < 0) {
         print_error("Unable to run sub-command.\n");
